crates/astria-sequencer/src/grpc/sequencer.rs

```diff
@@ -152,6 +160,53 @@ impl SequencerService for SequencerServer {
 
         Ok(Response::new(block))
     }
+
+    #[instrument(skip_all)]
+    async fn get_pending_nonce(
+        self: Arc<Self>,
+        request: Request<GetPendingNonceRequest>,
+    ) -> Result<Response<GetPendingNonceResponse>, Status> {
+        use astria_core::primitive::v1::Address;
+
+        use crate::accounts::state_ext::StateReadExt as _;
+
+        let request = request.into_inner();
+        let Some(address) = request.address else {
+            trace!("required field address was not set",);
```

Comment:
`info!` level. We want to know what happened when a request is rejected.

Reply:
that seems really high considering these endpoints could potentially be hit a lot, was concerned about spamming the logs, can change tho

Reply:
Don't be. The job of a service is to give a good idea about what's going on during its execution. Obviously within reason - you don't want to emit events in a hot loop.

It's the job of the observability platform to make sense of the traces/events that it collects by either dropping them or displaying them such that they are easy to understand.

Reply:
okay, updated!